src/providers/journal/journal.spec.ts

```diff
@@ -15,22 +16,22 @@ describe('PROVIDER: journal - ', () => {
   };
 
   beforeEach(() => {
-    provider = new JournalProvider(httpMock as any, dateTimeUtilMock);
+    provider = new JournalProvider(httpMock as any, dateTimeUtilMock, new AppConfigProvider());
```

Comment:
We should mock out AppConfigProvider for unit tests

Reply:
done